app/components/admin/add_lead_providers_form_component.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= form_with url: form_url, method: :post, local: true do |f| %>
+  <%= f.hidden_field :year, value: year %>
+  <%= f.hidden_field :page, value: page %>
+  <%= f.hidden_field :q, value: q %>
+
+  <% if current_partnerships.any? %>
+    <div class="govuk-inset-text">
+      <h3 class="govuk-heading-s">Currently working with:</h3>
+      <ul class="govuk-list govuk-list--bullet">
+        <% current_partnership_names.each do |name| %>
+          <li><%= name %></li>
+        <% end %>
+      </ul>
```

Comment:
We can use [the list helper](https://govuk-components.netlify.app/helpers/list/#bulleted-lists) here:

```suggestion
      <%= govuk_list(current_partnership_names.map(&:name), type: :bullet) %>
```